### ISSUE TITLE
Design adjustments

### ISSUE THREAD

**Issue by michaelbrooks** [hds-lab/textvisdrg#119]
A current screenshot is at the bottom.

- We might want to move the research questions up on the UI
- How can we change the appearance of dimension inventory/dropzones to make the interaction more discoverable?
- If we want to click on a dimension and see article info, what should it look like? (example article info is below)
- How can we adjust the style of the example messages to make it more readable and inviting?

```json
[{  
    "id":21,
    "authors":"Adel M. Aladwani",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/Aladwani_2015_Facilitators, characteristics, and impacts of Twitter use.pdf",
    "title":"Facilitators, characteristics, and impacts of Twitter use: Theoretical analysis and empirical illustration",
    "year":2015,
    "venue":"International Journal of Information Management"
},
{  
    "id":35,
    "authors":"Deen Freelon\nDavid Karpf",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/Freelon_Karpf_2015_Of big birds and bayonets.pdf",
    "title":"Of big birds and bayonets: hybrid Twitter interactivity in the 2012 Presidential debates",
    "year":2015,
    "venue":"Information, Communication &amp; Society"
},
{  
    "id":55,
    "authors":"Evelien D'heer\nPieter Verdegem",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/D'heer_Verdegem_2015_What social media data mean for audience studies.pdf",
    "title":"What social media data mean for audience studies: a multidimensional investigation of Twitter use during a current affairs TV pr",
    "year":2015,
    "venue":"Information, Communication &amp; Society"
},
{  
    "id":56,
    "authors":"Shaomei Wu\nJake M. Hofman\nWinter A. Mason\nDuncan J. Watts",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/Wu et al_2011_Who says what to whom on twitter.pdf",
    "title":"Who says what to whom on twitter",
    "year":2011,
    "venue":"Proceedings of the 20th international conference on World wide web"
},
{  
    "id":22,
    "authors":"Katrin Weller\nAxel Bruns",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/Weller_Bruns_2013_Friends or followers.pdf",
    "title":"Friends or followers? The relations between German soccer clubs and their fans on Twitter",
    "year":2013,
    "venue":"Selected Papers of Internet Research"
},
{
    "id":37,
    "authors":"S. Stieglitz\nLinh Dang-Xuan",
    "link":"http://nanro.org/~nanchen/social_science_papers/pdfs/Stieglitz_Dang-Xuan_2012_Political Communication and Influence through Microblogging–An Empirical.pdf",
    "title":"Political Communication and Influence through Microblogging–An Empirical Analysis of Sentiment in Twitter Messages and Retweet B",
    "year":2012,
    "venue":"2012 45th Hawaii International Conference on System Science (HICSS)"
}]
```

![image](https://cloud.githubusercontent.com/assets/1330400/6543485/618bd132-c4d2-11e4-87e8-6344e1699f67.png)

**Comment by michaelbrooks** [comment]
Specs from Ray:

1. Research Questions area went up and now is located at the upper side of visualization.
2. We added drag&drop zone at the x and y axis and added the instruction (check image resource at Google drive) at the left-bottom side from the origin of X-Y plane. Drag and drop zone: 
  - Dimension undefined:
    Outer round-shaped rectangular color stroke (#999999 / width 3px / Dash array 3,5 / border-radius 40px) 
    Inner circle color stroke (#999999 / width 2px / Dash array 3,5)

  - Dimension defined, x axis:
    Outer round-shaped rectangular color stroke (#29D7FF / width 3px / Dash array 3,5 / border-radius 40px) fill (#29D7FF, opacity 30%)
  - Dimension defined, y axis:
    Outer round-shaped rectangular color stroke (#FF174E/ width 3px / Dash array 3,5 / border-radius 40px) fill (#FF174E, opacity 15%)

3. Now the left dimension selector has some updates:
  - Selector size changed: height 35px, width 200px
  - We put icons to present the category for each (i.e. Times / Contents / Interaction / Meta / Author, check image resource at Google drive)
  - Now there are bright and dark separators. Dark: #555, dash array 1,1 - inter-category separators / Bright: #999 - category separators Both separators have left-margin 5px; right-margin 5px;

4. Example tweets has some updates:
  - Now there are separators.  #555, dash array 1,1, left and right margin 5px

**Comment by michaelbrooks** [comment]
![image](https://cloud.githubusercontent.com/assets/1330400/6855128/edd5ee40-d3b2-11e4-8a29-d78a9413402e.png)

**Comment by michaelbrooks** [comment]
![image](https://cloud.githubusercontent.com/assets/1330400/6858318/c8d25a3a-d3ca-11e4-929d-f7a85e736e36.png)

**Comment by nanchenchen** [comment]
Cool! Is this the current looking?

**Comment by michaelbrooks** [comment]
No, that's Ray's mockup, but I have it 95% done. I'll push the changes soon.

**Comment by michaelbrooks** [comment]
Fixed in #158
